src/main/java/seedu/address/model/charge/Charge.java

```diff
@@ -0,0 +1,67 @@
+package seedu.address.model.charge;
+
+import static java.util.Objects.requireNonNull;
+
+/**
+ * Represents the daily amount chargeable of a Pet in the address book.
+ * Guarantees: details are present and not null, field values are validated, immutable.
+ */
+public class Charge {
+    public static final String MESSAGE_INVALID_CHARGE_FORMAT = "Charge should be formatted as [number] or "
+             + "[number1.number2] where number2 can be up to 2 decimal places!";
```

Comment:
This message can be clearer I feel, maybe include examples?

Reply:
Alrights!